tests/test_haydn_op20.py

```diff
@@ -0,0 +1,82 @@
+# -*- coding: utf-8 -*-
+import music21
+
+from mirdata.datasets import haydn_op20
+from tests.test_utils import run_track_tests
+
+
+def test_track():
+    default_trackid = "0"
+    data_home = "tests/resources/mir_datasets/haydn_op20"
+    dataset = haydn_op20.Dataset(data_home)
+    track = dataset.track(default_trackid)
+
+    expected_attributes = {
+        "humdrum_annotated_path": "tests/resources/mir_datasets/haydn_op20/op20n1-01.hrm",
+        "title": "op20n1-01",
+        "track_id": "0",
+    }
+
+    expected_property_types = {
+        "duration": int,
+        "chords": list,
+        "roman_numerals": list,
+        "keys": list,
+        "score": object,
```

Comment:
this one is a score object no? maybe indicate it like that?

Reply:
done here 
51441e1